axis.go

```diff
@@ -280,12 +280,10 @@ func (a *verticalAxis) size() (w vg.Length) {
 // draw draws the axis along the left side of a draw.Canvas.
 func (a *verticalAxis) draw(c draw.Canvas) {
 	x := c.Min.X
+	a.Label.TextStyle.Rotation += math.Pi / 2.
```

Comment:
It bothers me that calling `draw` has the side affect of Incrementing `Rotation`. Won't this cause a second call to draw to have an incorrect rotation? Can we do this without changing the fields of the axis? At very least, we should have a deferred call that restores the original rotation.